src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -0,0 +1,212 @@
+              <RadioGroup
+                {...field}
+                row
+                onChange={(event) => setFieldValue(field.name, JSON.parse(event.target.value))}>
+                <FormControlLabel
+                  value={false}
+                  control={<Radio />}
+                  label={t('registration.files_and_license.accepted')}
+                />
+                <FormControlLabel
+                  value={true}
+                  control={<Radio />}
+                  label={t('registration.files_and_license.published')}
+                />
+              </RadioGroup>
+              {error && touched && <FormHelperText error>{error}</FormHelperText>}
+            </FormControl>
+          )}
+        </Field>
+      </TableCell>
+
+      <TableCell>
+        <Field name={`${baseFieldName}.${SpecificFileFieldNames.EmbargoDate}`}>
+          {({ field, meta: { error, touched } }: FieldProps) => (
+            <Box sx={{ minWidth: '12rem' }}>
+              <DatePicker
+                {...field}
+                label={t('registration.files_and_license.file_publish_date')}
+                PopperProps={{
+                  'aria-label': t('registration.files_and_license.file_publish_date'),
+                }}
+                value={field.value ?? null}
+                onChange={(date) => setFieldValue(field.name, date ?? '')}
+                inputFormat="dd.MM.yyyy"
+                maxDate={new Date(new Date().getFullYear() + 5, 11, 31)}
+                mask="__.__.____"
+                disabled={file.administrativeAgreement}
+                renderInput={(params) => (
+                  <TextField
+                    {...params}
+                    data-testid={dataTestId.registrationWizard.files.embargoDateField}
+                    variant="filled"
+                    onBlur={() => !touched && setFieldTouched(field.name)}
+                    error={!!error && touched}
+                    helperText={<ErrorMessage name={field.name} />}
+                  />
+                )}
+              />
+            </Box>
+          )}
+        </Field>
+      </TableCell>
+
+      <TableCell>
+        <Field name={`${baseFieldName}.${SpecificFileFieldNames.License}`}>
+          {({ field, meta: { error, touched } }: FieldProps) => (
+            <TextField
+              id={field.name}
+              data-testid={dataTestId.registrationWizard.files.selectLicenseField}
+              sx={{ minWidth: '15rem' }}
+              select
+              SelectProps={{
+                renderValue: (option) => {
+                  const selectedLicense = licenses.find((license) => license.identifier === option);
+                  return selectedLicense ? (
+                    <Box sx={{ display: 'flex', alignItems: 'center', gap: '0.5rem' }}>
+                      <img style={{ width: '5rem' }} src={selectedLicense.logo} alt="" />
+                      <span>{t(`licenses.labels.${option}` as any)}</span>
```

Comment:
hm, skjer det noe dumt om man fjerner as any her?
```suggestion
                      <span>{t(`licenses.labels.${option}`)}</span>
```

Reply:
Argument of type '`licenses.labels.${string}`' is not assignable to parameter of type...

Reply:
ah, ja, sikkert fordi `option` er type `unknown` :(  la gå